app/controllers/api/api_controller.rb

```diff
@@ -11,5 +11,42 @@ def missing_params_error_response(exception)
         ]
       }
     end
+
+    def authenticate_user!
+      if current_user
+        @user = current_user
+      else
+        render json: {
+          status: 'error',
+          messages: ['You have no access to this content or action']
+        }
+      end
+    end
+
+    def single_track_playlist_response(track_playlist)
+      has_voted_up = current_user.voted_up_on? track_playlist # TODO: To be modified with current_user
```

Comment:
Maybe we can remove the comment now?

Reply:
absolutely no idea why it's still there, I'll get rid of it, thx